dask/dataframe/io/tests/test_parquet.py

```diff
@@ -1688,3 +1688,12 @@ def test_passing_parquetfile(tmpdir):
 
     # should pass, because no need to re-read metadata
     dd.read_parquet(pf)
+
+
+def test_pathlib_path(tmpdir):
```

Comment:
I recommend adding the `engine` pytest fixture here

```suggestion
def test_pathlib_path(tmpdir, engine):
```

Reply:
removed in favor of @rjzamora 's #4995